README.md

```diff
@@ -171,6 +172,24 @@ Passwordless.token_generator = -> (session) {
 
 Session is going to keep generating tokens until it finds one that hasn't been used yet. So be sure to use some kind of method where matches are unlikely.
 
+### Token and Session Expiry
+
+#### Token timeout
+
+By default, sign in tokens generated by Passwordless are made invalid after `1 hour` from the time they are generated. If you wish you can override this and supply your custom Proc function that will return a valid datetime object. Make sure the generated time is in the future.
+
+```ruby
+Passwordless.timeout_at = lambda { 2.hours.from_now }
```

Comment:
Good idea to make it a proc so we can still use Rails' helpers.

Should we make it even more explicit that people remember to use a proc/lambda? So they don't end up timing out all their sessions two hours after every restart.

Reply:
How do you mean by explicit? Also, isn't this token timeout, it won't be reset after the restart right? we only call this at the time of session creation.